### ISSUE TITLE
Configure :chrome but firefox browser launch(axe-core-capybara)

### ISSUE THREAD

**Issue by naoya7076** [dequelabs/axe-core-gems#235]
I set `chrome` option as AxeCapybara.configure.
Then, I check driver variable.(check below code)
**Options browser is `:firefox`**, although @page.app=:chrome.

<img width="1267" alt="image" src="https://user-images.githubusercontent.com/32981554/173845039-bfa20f50-d203-4295-a5ab-b68a948c9c1d.png">

And axe says "Unable to find Mozilla geckodriver"
```
Selenium::WebDriver::Error::WebDriverError: Unable to find Mozilla geckodriver. Please download the server from https://github.com/mozilla/geckodriver/releases and place it somewhere on your PATH. More info at https://developer.mozilla.org/en-US/docs/Mozilla/QA/Marionette/WebDriver What's the issue here.
```
![image](https://user-images.githubusercontent.com/32981554/173844170-34f1385c-dc0a-43a3-8ff4-9da7ea54adb8.png)

Then, download firefox and execute `@axe.page.visit "https://google.com"` and firefox browser is launched.

The below is all the code
```.rb
require 'pry'
require "rspec"
require 'axe-capybara'

describe "axe test" do
  it "is known to be inaccessible, should fail" do
    @axe = AxeCapybara.configure(:chrome) do
    end
    binding.pry
    expect(page).not_to nil
  end
end
```

**Comment by michael-siek** [comment]
Hi @naoya7076 it looks like this has been fixed in 4.8.0. If this is still an issue please feel free to reopen this ticket or a new ticket. 

Ref: https://github.com/dequelabs/axe-core-gems/pull/303

**Comment by padmavemulapati** [comment]
verified with the release-branch code base, an issue is not there. It's working with the configured browser. 

for chrome config:


https://github.com/dequelabs/axe-core-gems/assets/47409579/e3d23698-8157-4e5c-8954-0746c87b815b

For Firefox config:


https://github.com/dequelabs/axe-core-gems/assets/47409579/48f4d0aa-a36f-442d-a9ed-8b60181ce023



## Environment:
| **Label**                         | **Value** |
| -------------------------- | ------ |
| **Product**                | _axe-core-gems/rspec-cucumber_ |
|**Version** | _release-2024-02-07_codebase_ |
| **OS-Details**      | _MAC- Intel Core i7 - 11.6.8 && Windows 11 _ |
|**BrowserDetails**|_Chrome Version 121.0.6167.139 && Firefox 122.0.1_|
